20_classes_lab/01.py

```diff
@@ -1,6 +1,27 @@
-"""
-The following code assumes a class
-named Summer exists. 
-Complete the class so correct result is printed
-"""
-
+""" Class Summer: sums input numbers """
+class Summer(object):
+
+    def __init__(self):
+        self.sum = 0
+
+    def add(self, *nums):
+        for n in nums: self.sum += n

```

Comment:
Python already has a sum function - just for you: `self.sum += sum(nums)` (and that's also the reason you should use another name for the member variable - sum is a bit confusing)